ups/meas_extensions_psfex.cfg

```diff
@@ -4,8 +4,8 @@ import lsst.sconsUtils
 
 dependencies = {
     "required": [],
-    "buildRequired": ["afw", "boost_test", "daf_base", "meas_algorithms",
-                      "swig", "utils", "psfex"],
+    "buildRequired": ["afw", "boost_test", "daf_base", "fftw_float",
+"meas_base","meas_algorithms","swig", "utils", "psfex"],
```

Comment:
Added back, I had my editor width set too small and when I made the change it pushed it back and I didn't catch it.